Assets/HoloToolkit/SpatialMapping/Scripts/SpatialProcessing/SurfaceMeshesToPlanes.cs

```diff
@@ -309,36 +305,4 @@ private void SetPlaneVisibility(SurfacePlane surfacePlane)
             surfacePlane.IsVisible = ((drawPlanesMask & surfacePlane.PlaneType) == surfacePlane.PlaneType);
         }
     }
-
-#if UNITY_EDITOR
-    /// <summary>
-    /// Editor extension class to enable multi-selection of the 'Draw Planes' and 'Destroy Planes' options in the Inspector.
-    /// </summary>
-    [CustomEditor(typeof(SurfaceMeshesToPlanes))]
-    public class PlaneTypesEnumEditor : Editor
```

Comment:
I wasn't able to find it anywhere in this PR. The only added file I see is SupportedInputInfo.cs